src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  date,
+  text,
+}) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
```

Comment:
Better to use semantic tag `artical` here
```suggestion
  <article>
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
  <article/>
```